src/common/offline_transformations/src/pruning/shrink_weights.cpp

```diff
@@ -70,6 +70,103 @@ static bool maybe_adopt_reshape_node(std::shared_ptr<ov::Node> reshape, ngraph::
     return true;
 }
 
+static bool handle_variadic_split(const std::shared_ptr<ov::Node>& split) {
+    const auto axis_node = ngraph::as_type<ngraph::opset6::Constant>(split->get_input_node_ptr(1));
+    if (!axis_node)
```

Comment:
Looks like `handle_variadic_split` and `handle_split` functions have a lot of common code. Can we unify it?

Reply:
There is little commonality, but to unify it I'd need a function that returns multiple values and the status, so I'd prefer to keep it that way.